packages/template-chakra-storefront/src/components/product-view-modal/bundle.test.js

```diff
@@ -151,13 +156,13 @@ test.skip('bundle product view modal disables update button when child is out of
         expect(screen.getByText('Out of stock')).toBeInTheDocument()
     })
 })
-
+//TODO: fix this failing test
```

Comment:
This test keeps failing for some reason. We have a ticket to address that, and I am current in progress of doing that